src/main/kotlin/com/criptext/mail/androidui/criptextnotification/NotificationOpenMailbox.kt

```diff
@@ -44,15 +44,14 @@ class NotificationOpenMailbox(override val ctx: Context): CriptextNotification(c
         val pushData = data as PushData.OpenMailbox
         val builder = NotificationCompat.Builder(ctx, CHANNEL_ID_OPEN_EMAIL)
                 .setContentTitle(pushData.title)
-                .setContentText(pushData.body)
+                .setContentText(ctx.getLocalizedUIMessage(UIMessage(R.string.open_email_notification_text, arrayOf(data.subject))))
                 .setSubText(pushData.recipientId.plus("@${pushData.domain}"))
                 .setAutoCancel(true)
                 .setContentIntent(clickIntent)
                 .setGroup(ACTION_OPEN)
                 .setGroupSummary(false)
                 .setSmallIcon(R.drawable.push_icon)
                 .setColor(Color.CYAN)
-                .setStyle(NotificationCompat.BigTextStyle().bigText(pushData.body))
```

Comment:
this is necessary for subjects too large.

Reply:
Ok